kolibri/auth/api.py

```diff
@@ -72,7 +72,7 @@ class FacilityUserViewSet(viewsets.ModelViewSet):
 
     def create(self, request, *args, **kwargs):
         try:
-            return super(viewsets.ModelViewSet, self).create(request, *args, **kwargs)
+            return super(FacilityUserViewSet, self).create(request, *args, **kwargs)
```

Comment:
I needed to properly call the parent method of FacilityUserViewSet class. But I was calling the parent method of ModelViewSet. Seemed to work still, but it was not correct format. Its a python 2 thing ;)